assets/newspack-ui/scss/elements/forms/_buttons.scss

```diff
@@ -1,16 +1,12 @@
 .newspack-ui {
 	// TODO: define some kind of size hierachy for buttons eg. small, medium, large;
 	// not needed for RAS, but likely for My Account.
-	.button,
-	button,
-	input[type='button'],
-	input[type='reset'],
-	input[type='submit'] {
+	&__button {
```

Comment:
I think it'd be good to have a second opinion from @laurelfulford about this change. I assume these selectors help enforce the button style on different scenarios, especially Woo-related. Maybe we move that to the `woocommerce` directory when necessary?

Personally, I like this change.

Reply:
To minimise surprise, we should be selective with styles. If the button styles should apply to elements not rendered by Newspack (e.g. Woo), I'd argue for a whitelist approach – explicitly apply the styles to Woo elements.